db/re/item_db_etc.yml

```diff
@@ -49516,8 +49518,9 @@ Body:
     Script: |
       bonus bCritAtkRate,15;
       if (getskilllv("AS_KATAR") >= 10) {
-         skill "TF_DOUBLE",3;
-         bonus bDoubleRate,15;
+         .@bonus = max(getskilllv("TF_DOUBLE"), 3);
+         skill "TF_DOUBLE",.@bonus;
+         bonus bDoubleRate,.@bonus*7;
```

Comment:
```suggestion
         skill "TF_DOUBLE",3;
         bonus bDoubleRate,21;
```

Reply:
I will withhold your suggestions because all these are defined in iteminfo.lub to check skill level of TF_DOUBLE.